tests/test_src.toml

```diff
@@ -0,0 +1,10 @@
+[tool.black]
+verbose = 1
+--check = "no"
+diff = "y"
+color = true
+line-length = 79
+target-version = ["py36", "py37", "py38"]
+exclude='\.pyi?$'
+include='\.py?$'
```

Comment:
Maybe reduce the configuration here too since it's only for testing the src argument.